.github/workflows/actions.yml

```diff
@@ -349,6 +349,9 @@ jobs:
         with:
           path: src
           submodules: recursive
+      - name: Set Xcode to 13.2.1
+        run: |
+          sudo xcode-select --switch /Applications/Xcode_13.2.1.app
```

Comment:
Nice idea. But can you select **Xcode 13.1** instead? We want this pipeline to test the oldest config possible.
Our project is for "macOS 10.14.4 or newer" + "Xcode 11.3.1 or newer", so configuring the pipeline to Xcode 13.1 (oldest configurable on GitHub) would be desirable.

Reply:
This is a little bit out of scope tbh.
For now, I just set Xcode to the version that was already used as default with macos-11 runner.